tests/simulators/carla/test_carla.py

```diff
@@ -8,6 +9,412 @@
+@pytest.mark.parametrize("modelName", benchModels)
+def test_bench_blueprints(getCarlaSimulator, launchCarlaServer, modelName):
+    pytest.importorskip("carla")
+    simulator, town, mapPath = getCarlaSimulator("Town01")
+    model_blueprint(simulator, mapPath, town, "Bench", modelName)
+
+
+from scenic.simulators.carla.blueprints import barrelModels
+
+
+@pytest.mark.skip(reason="Skipping test due to Carla memory leak issues")
+@pytest.mark.parametrize("modelName", barrelModels)
+def test_barrel_blueprints(getCarlaSimulator, launchCarlaServer, modelName):
+    pytest.importorskip("carla")
+    simulator, town, mapPath = getCarlaSimulator("Town01")
+    model_blueprint(simulator, mapPath, town, "Barrel", modelName)
+
+
+from scenic.simulators.carla.blueprints import atmModels
+
+
+@pytest.mark.skip(reason="Skipping test due to Carla memory leak issues")
+@pytest.mark.parametrize("modelName", atmModels)
+def test_atm_blueprints(getCarlaSimulator, launchCarlaServer, modelName):
+    pytest.importorskip("carla")
+    simulator, town, mapPath = getCarlaSimulator("Town01")
+    model_blueprint(simulator, mapPath, town, "ATM", modelName)
+
+
+from scenic.simulators.carla.blueprints import kioskModels
+
+
+@pytest.mark.skip(reason="Skipping test due to Carla memory leak issues")
+@pytest.mark.parametrize("modelName", kioskModels)
+def test_kiosk_blueprints(getCarlaSimulator, launchCarlaServer, modelName):
+    pytest.importorskip("carla")
+    simulator, town, mapPath = getCarlaSimulator("Town01")
+    model_blueprint(simulator, mapPath, town, "Kiosk", modelName)
+
+
+from scenic.simulators.carla.blueprints import ironplateModels
+
+
+@pytest.mark.skip(reason="Skipping test due to Carla memory leak issues")
+@pytest.mark.parametrize("modelName", ironplateModels)
+def test_iron_plate_blueprints(getCarlaSimulator, launchCarlaServer, modelName):
+    pytest.importorskip("carla")
+    simulator, town, mapPath = getCarlaSimulator("Town01")
+    model_blueprint(simulator, mapPath, town, "IronPlate", modelName)
+
+
+from scenic.simulators.carla.blueprints import trafficwarningModels
+
+
+@pytest.mark.skip(reason="Skipping test due to Carla memory leak issues")
+@pytest.mark.parametrize("modelName", trafficwarningModels)
+def test_traffic_warning_blueprints(getCarlaSimulator, launchCarlaServer, modelName):
+    pytest.importorskip("carla")
+    simulator, town, mapPath = getCarlaSimulator("Town01")
+    model_blueprint(simulator, mapPath, town, "TrafficWarning", modelName)
+
+
+from scenic.simulators.carla.blueprints import walkerModels
+
+
+@pytest.mark.skip(reason="Skipping test due to Carla memory leak issues")
+@pytest.mark.parametrize("modelName", walkerModels)
+def test_pedestrian_blueprints(getCarlaSimulator, launchCarlaServer, modelName):
+    pytest.importorskip("carla")
+    simulator, town, mapPath = getCarlaSimulator("Town01")
+    model_blueprint(simulator, mapPath, town, "Pedestrian", modelName)
+
+
+@flaky(max_runs=5, min_passes=1)
```

Comment:
**Note:** Introducing `flaky` to re-run dynamic scenario tests multiple times for `CARLA`. Reason being is that when starting the CARLA server via a subprocess via the `launchCarlaServer` pytest fixture, it could be segfault due to the memory leak errors that CARLA is currently experiencing.